scss/_card.scss

```diff
@@ -91,6 +91,10 @@
 // Background variations
 //
 
+.card-default {
+  background-color: #fff;
```

Comment:
This ought to use a variable, although I don't know what one would be appropriate. mdo should be able to provide guidance when he reviews this.